migrations/20160111211907.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE images ALTER COLUMN file_type TYPE VARCHAR(16);
```

Comment:
This was a bit of an error. Apparently sqlite ignore the length of a varchar, but postgresql doesnt. We tried to save content as `image/jpg` here, which is 9 characters. This caused an overflow and insert failed